tests/README.md

```diff
@@ -258,8 +258,6 @@ Directory structure is as follows:
 - `scripts`: contain scripts used to set up the environment and run the tests
 - `suite`: contains the test files
 
-> Note: Existing NFR tests will be migrated into this testing `suite` and results stored in the `results` directory.
-
```

Comment:
we still have a this [test](https://github.com/nginxinc/nginx-gateway-fabric/issues/1367) to be moved to `suite` so this note is still valid.

Reply:
Since it's in progress right now, I figured we don't really need this note here anymore.